Assets/Scripts/Models/Furniture.cs

```diff
@@ -20,7 +20,7 @@
 /// InstalledObjects are things like walls, doors, and furniture (e.g. a sofa).
 /// </summary>
 [MoonSharpUserData]
-public class Furniture : IXmlSerializable, ISelectable, IContextActionProvider
+public partial class Furniture : IXmlSerializable, ISelectable, IContextActionProvider
```

Comment:
This project is about getting people involved with game programming, C# and Unity through a tile-based base building game. Quill has stayed away from several concept that he thought would be off-putting to new users and `partial` might very well be one of them. I don't know of it being used anywhere else.

In order to start using it, we probably need to discuss the potential value.